pkg/oci/client/config.go

```diff
@@ -35,9 +38,16 @@ type AuthConfig struct {
 	UseInstancePrincipals bool   `yaml:"useInstancePrincipals"`
 }
 
+// VolumeProvisionerOptions describes the potential options available to users of the
+// volume provisioner
+type VolumeProvisionerOptions struct {
+	DisableRounding bool `yaml:"disableRounding"`
```

Comment:
Presuming we can change to having rounding disabled by default `RoundVolumeSize` would be my choice.